docker/Dockerfile

```diff
@@ -34,13 +34,15 @@ RUN apt-get update && apt-get install -y locales && rm -rf /var/lib/apt/lists/*
     && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 ENV LANG en_US.utf8
 
-# Install foundry
+# Install foundry from specific commit
 SHELL ["/bin/bash", "-c"]
-RUN curl -L https://foundry.paradigm.xyz | bash && \
+RUN curl -L https://raw.githubusercontent.com/foundry-rs/foundry/70d00222c2ef74fc484b8d98c5705d131ab31871/foundryup/install > /tmp/foundry-install-script && \
+    sed -i 's/\/foundry-rs\/foundry\/master\/foundryup/\/foundry-rs\/foundry\/70d00222c2ef74fc484b8d98c5705d131ab31871\/foundryup/g' /tmp/foundry-install-script && \
+    cat /tmp/foundry-install-script | bash && \
     echo "export PATH=\"$PATH:/$HOME/.foundry/bin\"">> ~/.bashrc && \
     source ~/.bashrc && \
     export PATH=$PATH:$HOME/.foundry/bin:$HOME/.cargo/bin && \
-    foundryup
+    foundryup --version nightly-70d00222c2ef74fc484b8d98c5705d131ab31871
```

Comment:
I believe the foundry nightly builds are regularly pruned, but some stick around longer tem. Do we know if this build is one of the long term ones?

Reply:
Good point. We might want to use the oct 2 release which is `5be158b`

Reply:
That's a great call out. If foundry isn't going have official release versions, I'd be a fan of keeping our own "stable" build hosted some where to ensure availability. 

Created an issue to track it here: https://github.com/ava-labs/teleporter/issues/88

The release used in this branch was just published today, so I think it will be okay to use for at least the short term.

Reply:
This should be fine if we merge in https://github.com/ava-labs/teleporter/pull/84, which will make it easy to spot the reason for failure if this version gets pruned.

Reply:
Just note that currently the oldest nightly build isn't a monthly release and hasn't been pruned yet is 4 days old.

Reply:
Jokes on us, this version has already been pruned 🙃